website/content/en/pre-docs/reference/provisioner-crd.md

```diff
@@ -35,21 +35,126 @@ spec:
-    - key: "kubernetes.io/arch" # If not included, all architectures are considered
+    - key: "kubernetes.io/arch" # If not included, reference cloud provider
       operator: In
       values: ["arm64", "amd64"]
-    - key: "kubernetes.io/os" # If not included, all operating systems are considered
+    - key: "kubernetes.io/os" # only linux is currently supported
       operator: In
       values: ["linux"]
-    - key: "karpenter.sh/capacity-type" # If not included, the webhook for the AWS cloud provider will default to on-demand
+    - key: "karpenter.sh/capacity-type" # If not included, reference the cloud provider
       operator: In
       values: ["spot", "on-demand"]
   # These fields vary per cloud provider, see your cloud provider specific documentation
   provider: {}
 ```
+
+## spec.requirements
+
+Kubernetes defines the following [well known labels]([[link]]), and cloud providers (e.g., AWS) implement them. They are defined at the "spec.requirements" section of the provisioner CRD. 
+
+These well known labels may be specified at the provisioner level, or in a workload definition (e.g., nodeSelector on a pod.spec). If the two conflict, the provisioner controls. In other words, a podspec may not override the provisioner with respect to constraints. 
+
+For example, the instance type may be specified using a nodeSelector in a workload definition. If the instance type requested at the workload (e.g., pod) level is not included in the provisioner list, karpenter will not create a node or schedule the pod. 
+
+📝 None of these values are required. 
+
+### Instance Types
+
+- key: `topology.kubernetes.io/instance-type`
+
+Generally, instance types should be a list and not a single value. Leaving this field undefined is recommended, as it maximizes choices for efficiently placing pods. 
+
+☁️ **AWS**
+
+Review [AWS instance types](https://aws.amazon.com/ec2/instance-types/).
+
+The default value includes all instance types with the exclusion of metal
+(non-virtualized),
+[non-HVM](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/virtualization_types.html),
+and GPU instances.
+
+View the full list of instance types with `aws ec2 describe-instance-types`.
+
+**Example**
+
+*Set Default with provisioner.yaml*
+
+```yaml
+spec:
+  requirements:
+    - key: node.kubernetes.io/instance-type
+      operator: In
+      values: ["m5.large", "m5.2xlarge"]
+```
+
+*Override with workload manifest (e.g., pod)*
+
+```yaml
+spec:
+  template:
+    spec:
+      nodeSelector:
+        node.kubernetes.io/instance-type: m5.large
+```
+
+### Availability Zones
+
+- key: `topology.kubernetes.io/zone`
+- value example: `us-east-1c`
+
+☁️ **AWS**
+
+- value list: `aws ec2 describe-availability-zones --region <region-name>`
+
+Karpenter can be configured to create nodes in a particular zone. Note that the Availability Zone us-east-1a for your AWS account might not have the same location as us-east-1a for another AWS account.
```

Comment:
```suggestion
Karpenter can be configured to create nodes in a particular zone. Note that the Availability Zone `us-east-1a` for your AWS account might not have the same location as `us-east-1a` for another AWS account.
```
I'm not sure if I've been consistent, but I generally like to mark literal values, such as `us-east-1a`, with backticks. I leave that to you.